lib/widgets/action_sheet.dart

```diff
@@ -315,8 +324,7 @@ class QuoteAndReplyButton extends MessageActionSheetMenuItemButton {
     // This will be null only if the compose box disappeared after the
     // message action sheet opened, and before "Quote and reply" was pressed.
     // Currently a compose box can't ever disappear, so this is impossible.
-    ComposeBoxController composeBoxController =
-      MessageListPage.ancestorOf(messageListContext).composeBoxController!;
+    var composeBoxController = findMessageListPage().composeBoxController!;
```

Comment:
Is `var` used for fitting this in one line?

Reply:
More that specifying the type feels redundant. It'd look like this:

Well, the alternative is:
```dart
    ComposeBoxController composeBoxController =
      findMessageListPage().composeBoxController!;
```

(or the same with `findMessageListPage()` moved before the newline).

Our usual practice is to say `final` and let the type be inferred. Here it's not `final` because it gets updated below; so `var` is the choice that corresponds to that.